src/common/snippets/src/op/brgemm.cpp

```diff
@@ -39,7 +39,7 @@ Brgemm::Brgemm(const Output<Node>& A, const Output<Node>& B,
     set_input_offset(offset_a, 0);
     set_input_offset(offset_b, 1);
     set_output_offset(offset_c, 0);
-    compute_block_size_values(blk_size_m, blk_size_k, blk_size_n);
+    set_block_size_values(blk_size_m, blk_size_k, blk_size_n);
```

Comment:
Yeah, I see your point. In fact, I tried exactly that, but it didn't worked out, because BrgemmCPU uses [default constructor op::Brgemm()](https://github.com/openvinotoolkit/openvino/blob/c4744e5adf9b46e0faf75a48d5f18825e4892143/src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_cpu.cpp#L69), so we still need this function anyway.
It really looks like an obsolete behavior and we should definitely refactor it in the future, but we have enough on our plate in this PR, so I suggest to do it in a separate activity.